shared/utils/relabel-ids.py

```diff
@@ -339,6 +337,15 @@ def verify_correct_form_of_referenced_cce_identifiers(xccdftree):
                 identcce.getparent().remove(identcce)
 
 
+def create_parser():
+    parser = ap.ArgumentParser(
+        description="This script finds check-content files (currently, "
+        "OVAL and OCIL) referenced from XCCDF and synchronizes all IDs.")
+    parser.add_argument("xccdf_file")
+    parser.add_argument("id_name", help="ID naming scheme")
+    return parser
+
+
 def main():
     if len(sys.argv) < 3:
```

Comment:
I think you can move the responsibility for this to argparse as well if you like.